qa/rpc-tests/p2p-fullblocktest.py

```diff
@@ -28,62 +43,67 @@ def __init__(self, tx = CTransaction(), n = -1):
 MAX_BLOCK_SIZE = 2000000
 MAX_BLOCK_SIGOPS_PER_MB = 20000
 
+
 class FullBlockTest(ComparisonTestFramework):
 
-    ''' Can either run this test as 1 node with expected answers, or two and compare them. 
+    ''' Can either run this test as 1 node with expected answers, or two and compare them.
         Change the "outcome" variable from each TestInstance object to only do the comparison. '''
+
     def __init__(self):
         self.num_nodes = 1
         self.block_heights = {}
         self.coinbase_key = CECKey()
         self.coinbase_key.set_secretbytes(b"horsebattery")
         self.coinbase_pubkey = self.coinbase_key.get_pubkey()
-        self.block_time = int(time.time())+1
+        self.block_time = int(time.time()) + 1
         self.tip = None
         self.blocks = {}
 
     def setup_network(self):
         self.nodes = start_nodes(1, self.options.tmpdir,
-                extra_args = [['-whitelist=127.0.0.1', '-debug', '-blocksizeacceptlimit=2']])
+                                 extra_args=[['-whitelist=127.0.0.1', '-debug', '-blocksizeacceptlimit=2']])
 
     def run_test(self):
         test = TestManager(self, self.options.tmpdir)
         test.add_all_connections(self.nodes)
-        NetworkThread().start() # Start up network handling in another thread
+        # Start up network handling in another thread
+        NetworkThread().start()
         test.run()
 
     def add_transactions_to_block(self, block, tx_list):
-        [ tx.rehash() for tx in tx_list ]
+        [tx.rehash() for tx in tx_list]
         block.vtx.extend(tx_list)
         block.hashMerkleRoot = block.calc_merkle_root()
         block.rehash()
         return block
-    
+
     # Create a block on top of self.tip, and advance self.tip to point to the new block
     # if spend is specified, then 1 satoshi will be spent from that to an anyone-can-spend output,
     # and rest will go to fees.
     def next_block(self, number, spend=None, additional_coinbase_value=0, script=None, block_size=0):
```

Comment:
Is the tool saying that there are too many arguments? The method needs them, and for a testing setup I am not all that worried about the API.